app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+  <div class="max-w-md w-full space-y-8">
+    <div>
+      <%= image_tag 'logo.svg', alt: 'LNU logo', width: 200, height: 200, class: 'mx-auto h-12 w-auto' %>
```

Comment:
This code is duplicated between views registrations/new and sessions/new. Should be pulled out to it's own little thing?